[New] Open mic announcement for 2024-05-30 (#129)
kbss-cvut/kbss-website: _drafts/2024-05-30-lessons-learned-from-the-latest-aviation-project-announcement.markdown

---

> title: "Lessons Learned from the Latest Aviation Project"

Reviewer: This is too vague for me. Who will remember/know which was the "Latest Aviation Project" in a couple of months?

Author: Lessons Learned from the Development of IS for Safety and Reliability of Aircrafts

Reviewer: For example, or (System instead of IS) "Lessons Learned from the Development of a System for Safety and Reliability of Aircrafts"

> title: "Lessons Learned from the Latest Aviation Project"

Author: ```suggestion
title: "Lessons Learned from the Development of a System for Safety and Reliability of Aircrafts"
```

> title: "Lessons Learned from the Development of a System for Safety and Reliability of Aircrafts"

Author: ```suggestion
title: "Lessons Learned from the Development of a System for Safety and Reliability of Aircraft"
```

categories: [Open Mic Announcement, Open Mic]
---

> Content: The Open mic session starts on Friday, May 31, 2024 at [this link](https://meet.jit.si/open-mic-kbss). The speaker is [Miroslav Blaško](https://kbss.felk.cvut.cz/web/team#miroslav-blaško), leader of the KBSS research group. He will discuss the lessons learned from the latest aviation project, which involves developing an information system for AERO Vodochody AEROSPACE a.s (AVA). This system aims to collect and process operational data to enhance the safety and reliability of aircraft. A key objective of the operational data is to provide input for failure analysis using the Fault Tree Analysis (FTA) technique.

Author: ```suggestion
Content: The Open mic session starts on Friday, May 31, 2024 at 12:30 at [this link](https://meet.jit.si/open-mic-kbss). The speaker is [Miroslav Blaško](https://kbss.felk.cvut.cz/web/team#miroslav-blaško), leader of the KBSS research group. He will discuss the lessons learned from the latest aviation project, which involves developing an information system for AERO Vodochody AEROSPACE a.s (AVA). This system aims to collect and process operational data to enhance the safety and reliability of aircraft. A key objective of the operational data is to provide input for failure analysis using the Fault Tree Analysis (FTA) technique.
```

> ##### The abstract

Reviewer: Abstract (to be consistent with other posts)

Author: Sorry, we do not have any guidelines about that, so I am not sure what feels inconsistent.

Would something like this be more acceptable to you ?:

-----

The development of an information system for AERO Vodochody AEROSPACE a.s. introduced several new challenges related to development workflow and the use of various tools. This talk will describe the development workflow and the use of KBSS tools such as SPipes, SForms, and JOPA. It will cover issues related to project management with GitHub, design with Figma, service orchestration with Docker Compose, and integration of user management with Keycloak. Additionally, it will address challenges related refactoring RDF data and shaping RDF data within REST APIs. The session will conclude with insights into the use of conversational AI in the development process.

Reviewer: I just meant to write "Abstract" instead of "The abstract" :-D

> ##### The abstract

Author: ```suggestion
##### Abstract
```

> This presentation outlines the development workflow and tools utilized in creating an information system for AERO Vodochody AEROSPACE a.s. It will cover the application of KBSS tools such as SPipes, SForms, and JOPA and address the new challenges encountered during the project. Additionally, the presentation will delve into using GitHub for project management and git workflows, Figma for design, and Docker for orchestration, highlighting associated issues. Challenges related to integrating Keycloak for user management across multiple applications, refactoring RDF data, and shaping RDF data within REST APIs will also be discussed. The session will conclude with insights on using conversational AI in the development process.

Author: ```suggestion
The development of an information system for AERO Vodochody AEROSPACE a.s. introduced several new challenges related to development workflow and the use of various tools. This talk will describe the development workflow and the use of KBSS tools such as SPipes, SForms, and JOPA. It will cover issues related to project management with GitHub, design with Figma, service orchestration with Docker Compose, and integration of user management with Keycloak. Additionally, it will address challenges related to refactoring RDF data and shaping RDF data within REST APIs. The session will conclude with insights into the use of conversational AI in the development process.
```
